runtime/track_memory_map.c

```diff
@@ -388,6 +401,22 @@ void track_memory_map(pid_t pid, struct memory_map *map) {
       fprintf(stderr, "could not interpret syscall!\n");
     }
 
+    /* pick up signal marking IA2 init finished to start forbidding init-only operations */
+    if (event_marks_init_finished(event, &event_info)) {
+      if (!memory_map_mark_init_finished(map)) {
+        fprintf(stderr, "attempting to re-finish init! (rip=%p)\n", (void *)regs.rip);
+        return;
```

Comment:
Do we want to exit with error or something here?

Reply:
Yeah--in the runtime integration branch `track_memory_map` returns a `bool`. I'll take this into account when rebasing.

Reply:
Done.